mysql-test/suite/rocksdb/t/update_with_keys.test

```diff
@@ -27,7 +23,7 @@ DROP TABLE t1;
 --echo # RocksDB: skip the test for secondary UNIQUE keys.
 --disable_parsing
 --error ER_GET_ERRMSG
-CREATE TABLE t1 (a INT, b CHAR(8), pk INT AUTO_INCREMENT PRIMARY KEY, UNIQUE INDEX(a)) ENGINE=innodb;
+CREATE TABLE t1 (a INT, b CHAR(8), pk INT AUTO_INCREMENT PRIMARY KEY, UNIQUE INDEX(a)) ENGINE=INNODB;
```

Comment:
The testcase strongly suggests that InnoDB is not the intended engine here. OK to fix in a separate commit